static/compiler.js

```diff
@@ -516,6 +543,7 @@ define(function (require) {
                 });
             });
             this.decorations = this.outputEditor.deltaDecorations(this.decorations, ranges);
+            this.rawDecorations = ranges;
```

Comment:
I think I'm a bit confused as to how this works :) I'm a bit vague on how Monaco's decorations work, but I thought we'd need to combine both the (optional) numeric decoration with the line nums decorations under all configurations. I think I see that rawDecorations[0] is always the numeric decoration? I think I'd prefer keeping the two decorations sets separates, and then always combining them in the calls to `deltaDecorations`...but if this works I'm happy :) - I can always tweak it once I understand it better.